setup.py

```diff
@@ -438,6 +439,33 @@ def srcpath(name=None, suffix=".pyx", subdir="src"):
     "pandas/_libs/tslibs/src/datetime/np_datetime_strings.h",
 ]
 
+import sysconfig
+
+ext_libraries = [
```

Comment:
Nit but maybe change this name to internal_libraries? My first thought when seeing ext_libraries was something installed on the system